src/steps/generic.rs

```diff
@@ -60,6 +60,27 @@ pub fn run_flutter_upgrade(run_type: RunType) -> Result<()> {
     run_type.execute(&flutter).arg("upgrade").check_run()
 }
 
+pub fn run_go(run_type: RunType) -> Result<()> {
+    let go = utils::require("go")?;
+    let gopath = run_type.execute(&go).args(&["env", "GOPATH"]).check_output()?;
```

Comment:
Could use `std::env::var` here, unless i'm missing something.

Reply:
This is necessary since users can define their `GOPATH` in their `GOENV` file instead of exporting it in their shell (see `go help environment`).